ui/de-lib/src/main/java/org/iplantc/de/client/DEClientConstants.properties

```diff
@@ -2,4 +2,7 @@ fileUploadServlet = /de/secured/fileUpload
 fileDownloadServlet = /de/secured/fileDownload
 logoutUrl = j_spring_security_logout
 deHelpFile = https://pods.iplantcollaborative.org/wiki/display/DEmanual/
+notificationWS = /de/websocket/notifications
+sysmessageWS = /de/websocket/system-messages
+wsProtocol = ws://
```

Comment:
An automatic inspection like that sounds like it'd make sense. Most clearly user-driven that way (not that we let people use plain http in prod anyway)!